From 0211b37f150f35f8015e9455d84f0764bb1cf8b1 Mon Sep 17 00:00:00 2001
From: Tomi Ollila <tomi.ollila@iki.fi>
Date: Sun, 11 May 2014 00:36:43 +0300
Subject: [PATCH 1/3] emacs install: make sure all components to be installed
 are there

`make install-emacs` will copy $(emacs_sources), $(emacs_images) and
$(emacs_bytecode) to their target directories. $(emacs_bytecode) was
already a prerequisite of make install-emacs as these obviously needed
to be build. Until a while ago all of $(emacs_sources) was available
in the repository, but now it includes `notmuch-version.el` which
is generated. In the future we may have generated emacs images too.
---
 emacs/Makefile.local | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/emacs/Makefile.local b/emacs/Makefile.local
index dcb4995..c0d6b19 100644
--- a/emacs/Makefile.local
+++ b/emacs/Makefile.local
@@ -69,7 +69,7 @@ install: install-emacs
 endif
 
 .PHONY: install-emacs
-install-emacs:
+install-emacs: $(emacs_sources) $(emacs_images)
 	mkdir -p "$(DESTDIR)$(emacslispdir)"
 	install -m0644 $(emacs_sources) "$(DESTDIR)$(emacslispdir)"
 ifeq ($(HAVE_EMACS),1)
-- 
1.9.3

